package.json

```diff
@@ -32,24 +32,25 @@
     "url": "http://blog.millermedeiros.com/"
   },
   "devDependencies": {
-    "esprima": "1.0",
-    "mocha": "https://github.com/millermedeiros/mocha/tarball/latest",
     "chai": "1.4",
+    "esprima": "1.0",
     "glob": "3.1",
     "istanbul": "~0.1.23",
     "jshint": "~2.3.0",
+    "mocha": "https://github.com/millermedeiros/mocha/tarball/latest",
```

Comment:
Since this isn't going away, publish it to npm as `mocha-miller` or something like that?